docs/getting_started/working_in_github.md

```diff
@@ -0,0 +1,48 @@
+# Getting the code
+
+-   Create your own Fork (once) (in web ui)
+-   Clone your fork locally
+    -   git clone https://github.com/\<your_id>/connectedhomeip.git
+-   Add the Upstream as a Remote
+    -   git remote add upstream https://github.com/project-chip/connectedhomeip
+
+# Updating
+
+-   git pull upstream / git fetch upstream
+-   We use submodules with submodules
+    -   git submodule update -f --init --recursive
```

Comment:
I'm not sure if we need to specifically call this out here, but bootstrap also does update the submodules, right? Not sure if it is recursive though 🤔

Reply:
Removed this file.